apps/frontend/src/app/Components/Character/CharacterEditor/Content.tsx

```diff
@@ -347,3 +359,114 @@ function InTeam() {
     </Box>
   )
 }
+function RemoveLoadout({
+  teamCharId,
+  onDelete,
+  teamIds,
+}: {
+  teamCharId: string
+  teamIds: string[]
+  onDelete: () => void
+}) {
+  const database = useDatabase()
+  const {
+    name,
+    description,
+    buildIds,
+    buildTcIds,
+    customMultiTargets,
+    bonusStats,
+  } = database.teamChars.get(teamCharId)!
+
+  const [show, onShow, onHide] = useBoolState()
+  const onDeleteLoadout = useCallback(() => {
+    onHide()
+    onDelete()
+  }, [onDelete, onHide])
+  return (
+    <>
+      <Button color="error" onClick={onShow}>
+        <DeleteForeverIcon />
+      </Button>
+      <ModalWrapper
+        open={show}
+        onClose={onHide}
+        containerProps={{ maxWidth: 'md' }}
+      >
+        <CardThemed>
+          <CardHeader
+            title={
+              <span>
+                Delete Loadout: <strong>{name}</strong>?
+              </span>
+            }
+            action={
+              <IconButton onClick={onHide}>
+                <CloseIcon />
+              </IconButton>
+            }
+          />
+          <Divider />
+          <CardContent>
+            {description && (
+              <CardThemed bgt="dark" sx={{ mb: 2 }}>
+                <CardContent>{description}</CardContent>
+              </CardThemed>
+            )}
+            <Typography>
+              Deleting the Loadout will also delete all data:
+            </Typography>
+            <List sx={{ listStyleType: 'disc', pl: 4 }}>
```

Comment:
Do we wanna have tooltips for each to display the names for builds, multi-opt etc etc? If we want the user to have the proper context